frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_project_chroots.py

```diff
@@ -52,17 +53,17 @@ def to_build_config_dict(project_chroot):
     return config_dict
 
 
-def rename_fields(input):
+def rename_fields(input_md):
     replace = {
         "additional_repos": "repos",
         "additional_packages": "buildroot_pkgs",
         "additional_modules": "module_toggle",
     }
-    output = input.copy()
-    for from_name, to_name in replace.items():
-        if from_name not in output:
-            continue
-        output[to_name] = output.pop(from_name)
+    output = MultiDict()
+    for key in input_md.keys():
+        new_key = replace.get(key, key)
+        for value in input_md.getlist(key):
+            output.add(new_key, value)
```

Comment:
Note: This required a fix as it previously ate the inputs and shrinked multiple values into a one (the first one).